src/screens/Tab/ScheduleScreen.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
```

Comment:
This screen used to be CalendarScreen, just renamed to ScheduleScreen and moved it into Tab folder